Modules/xSharePoint/DSCResources/MSFT_xSPManagedMetadataServiceApp/MSFT_xSPManagedMetaDataServiceApp.psm1

```diff
@@ -62,6 +64,10 @@ function Set-TargetResource
             
 
             if ($params.ContainsKey("InstallAccount")) { $params.Remove("InstallAccount") | Out-Null }
+            if ($params.ContainsKey("ContentTypeHubUrl")) {
+                $params.Add("HubUri", $params.ContentTypeHubUrl)
```

Comment:
exactly, my bad. I can't read shit in Canberran English.

On Tue, Mar 22, 2016 at 2:20 PM, Brian Farnhill <notifications@github.com>
wrote:

> In
> Modules/xSharePoint/DSCResources/MSFT_xSPManagedMetadataServiceApp/MSFT_xSPManagedMetaDataServiceApp.psm1
> <https://github.com/PowerShell/xSharePoint/pull/231#discussion_r56932611>:
>
> > @@ -62,6 +64,10 @@ function Set-TargetResource
> >
> >
> >              if ($params.ContainsKey("InstallAccount")) { $params.Remove("InstallAccount") | Out-Null }
> > +            if ($params.ContainsKey("ContentTypeHubUrl")) {
> > +                $params.Add("HubUri", $params.ContentTypeHubUrl)
>
> You mean like I did in the MOF description field which goes in to the
> PowerShell help and the wiki documentation?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/PowerShell/xSharePoint/pull/231/files/a6a770ebaf00b5cc72992108b60f4be12840a5e1#r56932611>
>